src/components/pages/devices/devices.scss

```diff
@@ -8,3 +8,8 @@
   padding: 20px;
   @include rem-font-size(20px);
 }
+.device {
```

Comment:
`.device` should be nested in `.devices-container`. Every component with css elements should be encapsulated in a container class to avoid polluting the global css namespace.

Reply:
OK